pennylane/utils.py

```diff
@@ -144,11 +144,15 @@ def sparse_hamiltonian(H, wires=None):
 
     if wires is None:
         wires = H.wires
+    else:
+        wires = qml.wires.Wires(wires)
```

Comment:
Something I missed in a previous PR where the wires are now introduced as an argument...